src/logic/equiv/set.lean

```diff
@@ -100,13 +100,6 @@ preimage_eq_iff_eq_image e.bijective
 lemma eq_preimage_iff_image_eq {α β} (e : α ≃ β) (s t) : s = e ⁻¹' t ↔ e '' s = t :=
 eq_preimage_iff_image_eq e.bijective
 
-@[simp] lemma prod_comm_preimage {α β} {s : set α} {t : set β} :
-  equiv.prod_comm α β ⁻¹' t ×ˢ s = s ×ˢ t :=
-preimage_swap_prod _ _
-
-lemma prod_comm_image {α β} {s : set α} {t : set β} : equiv.prod_comm α β '' s ×ˢ t = t ×ˢ s :=
-image_swap_prod _ _
-
```

Comment:
Where did these go?

Reply:
They are useless now that we have `image_prod_swap` and `preimage_prod_swap`. The first one was notably not simp  normal form anymore.

Reply:
If only you had somewhere to explain this type of change when you make the PR so as not to make it look like an accident ;)

Reply:
Didn't get the chance! Have been busy today ;)